cibyl/sources/jenkins.py

```diff
@@ -499,90 +510,228 @@ def get_deployment(self, **kwargs):
 
         return AttributeDictValue("jobs", attr_type=Job, value=job_objects)
 
-    def add_job_info_from_artifacts(self, job: Dict[str, str]):
+    def add_job_info_from_artifacts(self, job: dict):
         """Add information to the job by querying the last build artifacts.
 
         :param job: Dictionary representation of a jenkins job
         :type job: dict
         """
-        possible_artifacts = ["artifacts/jp.env", ".sh/run.sh", ".envrc"]
         job_name = job['name']
-        artifact = None
-        for artifact_path in possible_artifacts:
-            artifact_url = f"job/{job_name}/lastBuild/artifact/{artifact_path}"
-            try:
-                artifact = self.send_request(item=artifact_url, query="",
-                                             api_entrypoint="",
-                                             raw_response=True)
-                break
-            except JenkinsError:
-                LOG.debug("Found no artifact %s for job %s", artifact_path,
-                          job_name)
-                continue
-
-        if artifact is None:
+        build_description = job["lastCompletedBuild"].get("description")
+        if build_description is None:
```

Comment:
`if not build_description:` will catch also empty descriptions

Reply:
Good catch, thanks